managers/urls.py

```diff
@@ -8,14 +8,10 @@
     url("^projects/(?P<pk>[0-9]+)/$", views.ProjectDetailView.as_view(), name="custom-project-detail"),
     url("^projects/(?P<pk>[0-9]+)/update/$", views.ProjectUpdateView.as_view(), name="custom-project-update"),
     url("^projects/(?P<pk>[0-9]+)/delete/$", views.ProjectDeleteView.as_view(), name="custom-project-delete"),
+    url("^project/task-activities/$", views.ManageTaskActivities.as_view(), name="project-task-activities"),
```

Comment:
You should change also name of this url. It isn't anymore `project-task-activities` it's something more like `manage-task-activities`.

Reply:
changed